certified-connectors/PDFco/apiDefinition.swagger.json

```diff
@@ -38,8 +26,8 @@
     "/v1/pdf/convert/from/html": {
       "post": {
         "responses": {
-          "200": {
-            "description": "OK, success",
+          "default": {
```

Comment:
it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all responses in the swagger file.

Reply:
@vmanoharas , I have modified response codes